README.md

```diff
@@ -1 +1,116 @@
 # python-esiclient
+
+`python-esiclient` extends the OpenStack CLI to provide simplified user workflows, encapsulating multiple OpenStack CLI commands into a single CLI command.
+
+## `openstack esi node network <command>`
+
+These commands manage network connections to nodes.
+
+### `openstack esi node network list`
+
+List node/network attachments.
+
+```
+   openstack esi node network list
+       [--node <node>]
+       [--network <network>]
+```
+
+**--node <node>**: Filter by node (name or UUID)
+**--network <network>**:  Filter by network (name or UUID)
+
+### `openstack esi node network attach`
+
+Attach network to a node.
+
+```
+   openstack esi node network attach
+       [--network <network>]
+       [--port <port>]
+       [--mac-address <mac-address>]
+       <node>
+```
+
+*Exactly one of network or port must be specified.*
```

Comment:
You normally show this sort of requirement using `( option1 | option2 )` in your help output, e.g:

```
openstack esi node network attach
  (--network <network> | --port <port>)
  [--mac-address <mac-address>]
  <node>
```

Reply:
Ah, fixed!